editions/2015-02.md

```diff
@@ -0,0 +1,61 @@
+
+This is a very active project. Get involved as a volunteer to help test, write for [the Growstuff blog](http://blog.growstuff.org/), [contribute code to the application](https://github.com/growstuff) or just dig in, grow some food and share your seeds and data.
+
+
+### [Nick Evershed injects some context into the news](http://www.theguardian.com/news/datablog/2015/feb/14/tony-abbott-is-not-alone-in-using-the-word-holocaust-to-score-political-points)
+
+After Tony Abbott apologised for saying Labor created a “holocaust” in defence industry jobs, [Nick Evershed](https://twitter.com/nickevershed) used the [OpenAustralia API](http://www.openaustralia.org.au/api/) to provide more context about the use of the term in Parliament.
+
+
+### [Hackfarming Blood Buddies](https://adactio.com/journal/8352)
+
+[Jeremy Keith](https://adactio.com/) takes us through the process of planning, building and deploying a hack to help more people donate blood at [Clearleft’s](http://clearleft.com/) Hackfarm getaway this year.
+
+Their 2012 Hackfarm resulted in a prototype for [a personal political opinion tracker](http://hackfarm.org/product)—a few people at the [OpenAustralia Foundation meetup](http://www.meetup.com/OpenAustralia-Foundation/) have suggested similar concepts for projects they want to work on.
+
+
+### [Local events for Internation Open Data Day](http://au.okfn.org/2015/02/10/australian-activities-on-international-open-data-day-21-february/)
+
+This Saturday, 21st Feb, is [International Open Data Day](http://opendataday.org/). The Open Knowledge Foundation Australia have done a wrap-up of events, including a day long workshopping session hosted by [Code for Australia](http://www.codeforaustralia.org/) in [Sydney tomorrow](http://www.eventbrite.com.au/e/international-open-data-day-sydney-tickets-15462067460). It is a free event and registrations are still open if you can make it.
+
+
+### [Learning to contribute with Git](http://www.git-tower.com/learn/)
+
+Contributing through the Git version control system is an important skill for people working on civic tech and open source generally—but it ain't that easy to learn and it's not always clear where to start. We've observed that it can be a barrier to people getting involved.
+
+Aside from the link above there are many tutorials and articles to help. Luke found the [Web Ahead Podcast's Git episode](http://thewebahead.net/40) very helpful. Github also have a whole [list of resources](https://help.github.com/articles/good-resources-for-learning-git-and-github/). Sometimes the best thing to do is ask someone to sit down and take you through it.
+
```

Comment:
How about 
'Once you've learned it you'll have superpowers, so don't be afraid to ask if you need help' @henare your thoughts?